.gitignore

```diff
@@ -24,3 +24,7 @@ package-lock.json
 /.node_modules.ember-try/
 /bower.json.ember-try
 /package.json.ember-try
+
+# Intellij
+.idea
+ember-metrics.iml
```

Comment:
IMHO this belongs in your global git ignore. https://stackoverflow.com/a/7335487

Reply:
Thanks, created. Dont want to update the .gitignore and cause the build to need to be kicked off again. Can fix this up if I have to update anything or just before the PR is accepted.